data/mods/Aftershock/items/weapons.json

```diff
@@ -55,21 +58,24 @@
     "id": "afs_energy_saber_on",
     "type": "TOOL",
     "copy-from": "afs_energy_saber_off",
-    "name": { "str": "energy saber (active)", "str_pl": "energy sabers (active)" },
-    "description": "This glittering blue saber of cyan light hums gently, extending from its superalloy handle.  Its edge is lethally sharp, and being made of light, it's weightless, and immune to being damaged.  It's not white-hot like in pop culture, so it can't cut through anything, but there's not much in the way of flesh that it won't slice through easily.",
-    "volume": "1 L",
+    "name": { "str": "CE-4 \"The Claw\" (Lethal)", "str_pl": "CE-4 \"The Claws\" (Lethal)" },
+    "description": "The CE-4, commonly known as \"The Claw\", is a popular self-defense weapon used by corporate enforcers everywhere, especially on dangerous planets like Salus IV.  Its most notable aspect is that it comes with two modes, one non-lethal and one lethal.  Currently, it is in its lethal configuration, looking like a sleek sword, humming softly as its blade glows from heat.",
+    "weight": "760 g",
+    "volume": "2 L",
+    "longest_side": "90 cm",
     "color": "light_blue",
+    "looks_like": "arming_sword",
     "turns_per_charge": 4,
     "relic_data": { "passive_effects": [ { "id": "afs_ench_esab" } ] },
     "to_hit": 2,
```

Comment:
Probably deserves some measure of bahing damage as well, you could reduce the heat damage added by afs_ench_esab to compensate

Reply:
The bashing damage is for the non-lethal mode, since the lethal one is sharp.